File: src/java/eu/sig/training/ch03/withmapandtypes/ItalianFlag.java
package eu.sig.training.ch03.withmapandtypes;

import java.awt.Color;
import java.util.Arrays;
import java.util.List;

// tag::ItalianFlag[]
public class ItalianFlag implements Flag {
    public List<Color> getColors() {
        return Arrays.asList(Color.GREEN, Color.WHITE, Color.RED);
    }
}
// end::ItalianFlag[]

File: src/csharp/eu/sig/training/ch03/withmapandtypes/ItalianFlag.cs
using System.Collections.Generic;
using System.Drawing;

namespace eu.sig.training.ch03.withmapandtypes
{

    // tag::ItalianFlag[]
    public class ItalianFlag : IFlag
    {
        public IList<Color> Colors
        {
            get
            {
                return new List<Color> { Color.Green, Color.White, Color.Red };
            }
        }
    }
    // end::ItalianFlag[]

}
